src/public/js/index.js

```diff
@@ -1,28 +1,17 @@
-import getTimeRemaining from './countdown';
+import request from './utils/request.js';
 
-const initializeClock = (minutes) => {
-  document.getElementById('start').style.display = 'none';
-  document.getElementById('stop').style.display = 'block';
-  const endtime = Date.now() + (minutes * 60 * 1000); // number of miliseconds until countdown finishes
-  const clock = document.getElementById('clockdiv');
-  const timeinterval = setInterval(() => {
-    const t = getTimeRemaining(endtime);
-    const seconds = t.seconds < 10 ? '0' + t.seconds : t.seconds;
-    const minutes = t.minutes < 10 ? '0' + t.minutes : t.minutes;
-    clock.innerHTML = minutes + ':' + seconds;
-    if (t.total < 1000) {
-      clearInterval(timeinterval);
-    }
-  }, 1000);
-  document.getElementById('stop').addEventListener('click', () => resetClock(10, timeinterval));
-};
+import timer from './timer.js';
+import login from './login.js';
+import error from './error.js';
 
-const resetClock = (minutes, timeinterval) => {
-  clearInterval(timeinterval);
-  document.getElementById('stop').style.display = 'none';
-  document.getElementById('start').style.display = 'block';
-  const clock = document.getElementById('clockdiv');
-  clock.innerHTML = minutes + ':' + '00';
-};
+request.get('/orgs')
+  .then((res) => {
+    if (res.status === 200) {
+      timer(res.response);
```

Comment:
if the response for orgs comes back with a 200 we want to allow the user to choose an organisation not show the timer. We can change this when we implement that functionality